src/db/seeders/test/trackgroup-seeder.js

```diff
@@ -138,27 +138,7 @@ module.exports = {
             VALUES ('f84a89e1-1e5c-47d5-a07e-9bfc64ef4ff0', 6, '58991f22-b172-48e4-8b27-e0a4c946f9b2', '2684ca14-9864-4377-aab9-7a471b1f8d14', NULL, NULL, '2022-09-28 17:31:59.858+00', '2022-09-28 17:31:59.858+00', NULL);
         INSERT INTO public.track_group_items (id, index, track_group_id, track_id, track_performers, track_composers, updated_at, created_at, deleted_at) 
             VALUES ('ed84cd42-8990-4d8e-a9f6-e00aaacdc2bb', 8, '58991f22-b172-48e4-8b27-e0a4c946f9b2', '374d19fc-18af-474d-8c19-debc993db991', NULL, NULL, '2022-09-28 17:31:59.861+00', '2022-09-28 17:31:59.861+00', NULL);
-      
-
-        INSERT INTO public.plays (id, track_id, user_id, event, created_at) VALUES (1, '706cff12-ba44-49f7-8982-98b3996a2919', '251c01f6-7293-45f6-b8cd-242bdd76cd0d', 1, '2022-09-28 17:31:59.805211+00');
```

Comment:
Turns out these were conflicting with the id count when creating a new play, and also made it harder to check what you were trying to check because it wasn't clear ho wit would add.